message NewPageEvent {
	required uint64 id = 1;
}